### DISCUSSION TITLE
Is MikroORM config file necessary ?

### DISCUSSION THREAD

**Discussion by gdamou** [mikro-orm/mikro-orm#4021] [Q&A] [answered]
Hi, 

I tried to remove mikro-orm.config.ts from my project and obviously the CLI stopped to work. 

I am making migration in my code already, is there any need to keep mikro-orm.config except for CLI to work ? Is it possible to make everything you can do with CLI but manually with code ?

I am still learning the ORM so I want to be sure that not using CLI at all will not remove some useful features later haha

Thanks for your responses !

**Comment by B4nan** [comment, marked as answer]
Yeah you can do everything programmatically, and the config is needed only for the CLI - and can be obviously reused so act as the single source of truth.
